core/src/main/java/com/yahoo/imapnio/async/data/QResyncSeqMatchData.java

```diff
@@ -0,0 +1,61 @@
+package com.yahoo.imapnio.async.data;
+
+import java.util.List;
+
+import javax.annotation.Nonnull;
+
+/**
+ * This class models the QRESYNC Sequence match data.
```

Comment:
RFC reference

Reply:
Formal Syntax section:
https://tools.ietf.org/html/rfc7162#section-7

select-param        =/  "QRESYNC" SP "(" uidvalidity SP
                       mod-sequence-value [SP known-uids]
                       [SP seq-match-data] ")"
                       ;; Conforms to the generic select-param
                       ;; syntax defined in [RFC4466].